src/abbreviationYAxisFn.js

```diff
@@ -0,0 +1,7 @@
+const abbreviationYAxisFn = abb => {
+  const doesAbbrContainTail = RegExp(`[gjpqy]`).test(abb);
```

Comment:
Could maybe use a comment to explain what is a "tail" and why does this need to be done.